### DISCUSSION TITLE
进门后卡进度时间过长

### DISCUSSION THREAD

**Discussion by dkdd2020** [Melledy/Grasscutter#2369] [Q&A]
如题，不仅如此，控制台窗口一直显示public Failed to Open: PublicKey.txt及private Failed to Open: PrivateKey.txt。请问该怎么解决啊？？？
![image](https://github.com/Grasscutters/Grasscutter/assets/105190397/8cf73f5f-f763-470a-ac7b-4ee5f4fedaf5)

**Comment by NotThorny** [comment]
Failed to open PublicKey.txt is normal and can be ignored, it isn't actually an error. Your Grasscutter cmd would have better info about why you are stuck after entering the door.

**Comment by Rush-525** [comment]
你应该发grasscutter.java的命令窗口信息，但是这个窗口正常是这样子的
<img width="1280" alt="image" src="https://github.com/Grasscutters/Grasscutter/assets/88147648/d0bda962-4952-414b-bf0f-946f25ad09d6">
而另一窗口正常是这样子的
<img width="867" alt="image" src="https://github.com/Grasscutters/Grasscutter/assets/88147648/a8a47dbd-25ba-4dcb-9dda-61f044272db9">
你试一下把补丁全部删了，再重新下载，注意：一般用PrivateKey.txt而用PublicKey.txt
即下载补丁重命名为version.dll，新建PublicKey.txt输入公钥，保存

**Reply by Rush-525** [reply]
https://github.com/34736384/RSAPatch
